cttportfolio.html

```diff
@@ -0,0 +1,95 @@
+<!doctype html>
+
+<html lang= "en">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+
+<head>
+	<meta charset="UTF-8">
+	<title>HTML Exercise 1</title>
+	<link rel="stylesheet" type="text/css" href="cttportfolio.css">
+	
+</head>
+<body>
+	<nav>
+		 <img src="smiley.gif" alt="Logo replacement" height="42" width="42"> 
```

Comment:
Indentation is slightly off here, also it's better to define the `width` and `height` within the CSS, not HTML. I've noticed you've done this for multiple `<img />` tags below too.